src/assets/arrow/Ic_upvote_off.js

```diff
@@ -1,16 +1,23 @@
-        fill={props.color || COLORS.gray400}
-      />
+    <Svg width="20" height="18" viewBox="0 0 20 18" fill="none" {...props}>
+      <G clipPath="url(#clip0_20118_29079)">
+        <Path
+          fillRule="evenodd"
+          clipRule="evenodd"
+          d="M7.25328 15.2973L7.30187 15.276C7.31902 15.2681 7.34067 15.2581 7.36663 15.2461C9.08044 14.5541 10.9899 14.5631 12.6986 15.2732L19.1378 17.9508C19.2579 18.0002 19.3899 18.0102 19.5158 17.9795C19.6417 17.9488 19.7552 17.8788 19.8405 17.7793C19.9258 17.6797 19.9789 17.5555 19.9923 17.4238C20.0056 17.2921 19.9787 17.1594 19.9152 17.044L10.5434 0.320806C10.4887 0.223441 10.4098 0.142567 10.3147 0.086332C10.2434 0.0441098 10.1648 0.0168975 10.0836 0.0059575C10.0555 0.00200333 10.027 0 9.99847 0C9.88868 0 9.78082 0.0296089 9.68577 0.0858436C9.59072 0.142078 9.51183 0.222952 9.45706 0.320318L0.0840528 17.044C0.0196997 17.1594 -0.00790218 17.2925 0.00508606 17.4247C0.0180743 17.5569 0.071007 17.6817 0.156522 17.7817C0.242037 17.8817 0.355884 17.952 0.482228 17.9827C0.608572 18.0134 0.741132 18.0031 0.861473 17.9531L7.25328 15.2973ZM6.60981 13.6108L3.30233 14.9873L10.0004 3.03935L14.4341 10.9483C13.3631 11.1395 12.2321 11.4614 11.1971 11.8037C9.09819 12.498 7.20518 13.3404 6.65214 13.5933C6.63802 13.5991 6.62391 13.6049 6.60981 13.6108ZM15.3746 12.6259L16.698 14.9866L13.3898 13.6109C13.0885 13.4857 12.7819 13.3777 12.472 13.2873C13.47 12.9833 14.4812 12.7347 15.3746 12.6259Z"
+          fill={props.color || COLORS.gray400}
+        />
+      </G>
+      <Defs>
+        <ClipPath id="clip0_20118_29079">
+          <Rect width="19.998" height="18" fill="white" transform="translate(0.00195312)" />
+        </ClipPath>
+      </Defs>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SVG component `IcUpvoteOff` has been significantly refactored. The viewBox, dimensions, and path of the SVG have been updated. Additionally, a clipPath has been added to the SVG. This could potentially change the appearance of the icon in your application. Make sure to test this thoroughly in all places where this icon is used to ensure it still appears as expected.

Also, the fill color now defaults to `COLORS.gray400` if no color prop is provided. Ensure that this default color fits well with the overall design of your application in cases where no color prop is provided.

```diff
- <Svg width="22" height="12" viewBox="0 0 22 22" fill="none" {...props}>
+ <Svg width="20" height="18" viewBox="0 0 20 18" fill="none" {...props}>
```

```diff
- <Path
-   fill-rule="evenodd"
-   clipRule="evenodd"
-   d="M7.61098 15.611L7.61008 15.6114L4.30389 16.9874L11.0012 5.04082L15.4344 12.9488C14.3635 13.14 13.2326 13.4619 12.1978 13.8042C10.0991 14.4984 8.20628 15.3407 7.6533 15.5935C7.63918 15.5993 7.62507 15.6051 7.61098 15.611ZM3.36436 19.3281L8.25438 17.2974L8.30296 17.276C8.3201 17.2681 8.34176 17.2581 8.36771 17.2462C10.0813 16.5542 11.9906 16.5632 13.6991 17.2732L19.123 19.5286L19.2144 19.5666L19.642 19.7444L19.6552 19.7499L20.1375 19.9505C20.2575 19.9999 20.3896 20.0099 20.5155 19.9792C20.6413 19.9485 20.7548 19.8785 20.8401 19.779C20.9254 19.6795 20.9785 19.5553 20.9918 19.4236C21.0052 19.2919 20.9783 19.1592 20.9148 19.0438L20.685 18.6338L20.6817 18.628L20.441 18.1987L20.3888 18.1056L18.1418 14.0973L17.3971 12.7682L11.5441 2.3226C11.4894 2.22525 11.4105 2.14438 11.3155 2.08815C11.2441 2.04594 11.1655 2.01873 11.0843 2.00779C11.0562 2.00383 11.0278 2.00183 10.9992 2.00183C10.8895 2.00183 10.7816 2.03144 10.6866 2.08766C10.5915 2.14389 10.5127 2.22476 10.4579 2.32211L1.61022 18.1085L1.55793 18.2018L1.31586 18.6337L1.31497 18.6353L1.086 19.0438C1.02165 19.1592 0.994052 19.2923 1.00704 19.4245C1.02003 19.5567 1.07295 19.6814 1.15846 19.7814C1.24396 19.8814 1.35779 19.9517 1.48412 19.9824C1.61045 20.0131 1.743 20.0028 1.86332 19.9528L2.34797 19.7511L2.3517 19.7496L2.78473 19.5693L2.87624 19.5313L3.36436 19.3281ZM16.3748 14.6263L17.698 16.9866L14.3902 15.6112L14.3898 15.611C14.0886 15.4858 13.7823 15.378 13.4724 15.2875C14.4703 14.9836 15.4814 14.735 16.3748 14.6263Z"
-   fill={props.color || COLORS.gray400}
- />
+ <G clipPath="url(#clip0_20118_29079)">
+   <Path
+     fillRule="evenodd"
+     clipRule="evenodd"
+     d="M7.25328 15.2973L7.30187 15.276C7.31902 15.2681 7.34067 15.2581 7.36663 15.2461C9.08044 14.5541 10.9899 14.5631 12.6986 15.2732L19.1378 17.9508C19.2579 18.0002 19.3899 18.0102 19.5158 17.9795C19.6417 17.9488 19.7552 17.8788 19.8405 17.7793C19.9258 17.6797 19.9789 17.5555 19.9923 17.4238C20.0056 17.2921 19.9787 17.1594 19.9152 17.044L10.5434 0.320806C10.4887 0.223441 10.4098 0.142567 10.3147 0.086332C10.2434 0.0441098 10.1648 0.0168975 10.0836 0.0059575C10.0555 0.00200333 10.027 0 9.99847 0C9.88868 0 9.78082 0.0296089 9.68577 0.0858436C9.59072 0.142078 9.51183 0.222952 9.45706 0.320318L0.0840528 17.044C0.0196997 17.1594 -0.00790218 17.2925 0.00508606 17.4247C0.0180743 17.5569 0.071007 17.6817 0.156522 17.7817C0.242037 17.8817 0.355884 17.952 0.482228 17.9827C0.608572 18.0134 0.741132 18.0031 0.861473 17.9531L7.25328 15.2973ZM6.60981 13.6108L3.30233 14.9873L10.0004 3.03935L14.4341 10.9483C13.3631 11.1395 12.2321 11.4614 11.1971 11.8037C9.09819 12.498 7.20518 13.3404 6.65214 13.5933C6.63802 13.5991 6.62391 13.6049 6.60981 13.6108ZM15.3746 12.6259L16.698 14.9866L13.3898 13.6109C13.0885 13.4857 12.7819 13.3777 12.472 13.2873C13.47 12.9833 14.4812 12.7347 15.3746 12.6259Z"
+     fill={props.color || COLORS.gray400}
+   />
+ </G>
+ <Defs>
+   <ClipPath id="clip0_20118_29079">
+     <Rect width="19.998" height="18" fill="white" transform="translate(0.00195312)" />
+   </ClipPath>
+ </Defs>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->